int8 markerID
---
float32[] unitNormal
bool hasMask
bool viable
bool warn
